domains/mc.baldeagle.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SniverDaBest",
+        "discord": "919385800364425256"
```

Comment:
Need email field or OWL field.

Reply:
Only email field is necessary though, OWL is optional.